### ISSUE TITLE
Import fails missing MERCHANT_ID

### ISSUE THREAD

**Issue by Fleschmc** [Kount/kount-ris-python-sdk#28]
When I try to import any of the class objects it fails because there is a MERCHANT_ID variable that is referenced but never assigned a value. Here is the traceback:
```---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-8-0a4c29d97c8f> in <module>
----> 1 from kount.client import Client
      2 from kount.config import SDKConfig
      3 from kount.request import Update

/usr/local/lib/python3.6/site-packages/kount/__init__.py in <module>
      9 
     10 from .version import VERSION
---> 11 from kount.config import SDKConfig
     12 
     13 __author__ = SDKConfig.SDK_AUTHOR

/usr/local/lib/python3.6/site-packages/kount/config.py in <module>
      9 from . import resources
     10 from .version import VERSION
---> 11 from .settings import CONFIGURATION_KEY, SDK_AUTHOR, SDK_MAINTAINER, MAINTAINER_EMAIL, DEFAULT_TIMEOUT
     12 
     13 __author__ = SDK_AUTHOR

/usr/local/lib/python3.6/site-packages/kount/settings.py in <module>
      9 TEST_API_URL = "API_URL" # provided by kount
     10 TEST_API_KEY = "API_KEY" # kount API Key
---> 11 TEST_MERCHANT_ID =  MERCHANT_ID # provided by kount

NameError: name 'MERCHANT_ID' is not defined
```

It doesn't seem like anyone else has had this issue so I'm not sure what I'm missing. Any guidance would be appreciated.

**Comment by ShermanKester** [comment]
If you contact your Kount Customer Success Manager, they can get you the assigned Merchant ID, which can then be set in the SDK.

**Comment by ShermanKester** [comment]
Yes.  There should be a place to to set the Merchant ID as well.  The Merchant ID, the API Key, the Config Key, and the endpoint URLs.

**Comment by cbporch** [comment]
I also ran into this.  Manually editing a settings.py file to fix a `NameError` thrown by the instantiation of a test variable feels like an unusual approach for a Python library.  As it stands now, `TEST_API_URL` and `TEST_API_KEY` are both set to default strings, and IMHO `TEST_MERCHANT_ID` seems like it should probably follow suit.

**Comment by dharp-kount** [comment]
Corrected with version 3.3.4
